test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -48,18 +48,18 @@ Describe "Certificate Provider tests" -Tags "CI" {
             param([string] $path)
             $expectedResolvedPath = Resolve-Path -LiteralPath $path
             $result = Get-Item -LiteralPath $path
-            $result | should not be null
```

Comment:
The old test logic here is bad. Correcting the null test is causing the tests to fail.

Reply:
The core issue here is that `Get-Item Cert:\` returns nothing but the test is written like it expects an object back. I am setting this test to pending for now.

Reply:
@KevinMarquette Please don't mess functional and style changes in future - this slows down the review very much. Open new issue and fix bugs in other PRs.

Here please revert line 47. I believe a bug is in line 50 - it should be `$result = Get-ChildItem -LiteralPath $path`.